Travel Scheduler/Views/AttractionCollectionCell.m

```diff
@@ -48,6 +68,14 @@ - (void)setImage
     }
 }
 
+- (void)adjustUILabelFrame
+{
+    CGRect frame = self.titleLabel.frame;
+    frame.origin.y = self.contentView.bounds.size.height - self.titleLabel.frame.size.height - 20;//pass the Y cordinate
+    frame.origin.x = 7;//pass the X cordinate
+    self.titleLabel.frame= frame;
+}
```

Comment:
Just curious why you created a new CGRect object to hold the frame? Can you not just do self.titleLabel.frame.origin.y = ... and self.titleLabel.frame.origin.x = ...? 

Also, not sure what the normal practice is but I usually squash it all into one line with self.titleLabel.frame = CGRectMake(x..., y..., width..., height..) or if that's really long, I do int yCoord = self.contentView.bounds.... and int xCoord = 7 and then self.titleLabel.frame = CGRectMake(xCoord, yCoord, width..., height...)